packages/hardhat-core/src/internal/cli/analytics.ts

```diff
@@ -126,7 +124,7 @@ export class Analytics {
       dp: `/task/${taskName}`,
 
       // Host name.
-      dh: "cli.buidler.dev",
+      dh: "cli.hardhat.dev",
```

Comment:
Not 100% sure about this, but I think it makes sense?

Reply:
We don't own this domain 😭💔😰

Reply:
```suggestion
      dh: "cli.usehardhat.com",
```